docs/pages/components/step-input.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Step Input
+id: step-input
+keywords: Step Input, Step, Input
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/step-input.html
+folder: components
+---
+
+The Step Input component is an opinionated composition of the Input, Button components.
+It is typically used to enter numbers.The value can be increased or reduced with the provided controls.
+{: .docs-intro}
+
+
+## Step Input with Cozy
+Cozy Mode is the default one, it should be used in mobile and tablet devices.
+
+{% capture step-input-cozy %}
+<div class="fd-step-input">
+        <button aria-label="Step down" class="
+            fd-button
+            fd-button--transparent
+            fd-step-input__button
+            sap-icon--less"
+            onclick="stepInputValue('step-3', 'down');"
+            type="button">
+        </button>
+    <input class="
+            fd-input 
+            fd-input--no-number-spinner 
+            fd-step-input__input
+    " id="step-3" type="number" value="0">
+        <button aria-label="Step up" class="
+            fd-button 
+            fd-button--transparent
+            fd-step-input__button
+            sap-icon--add"
+            onclick="stepInputValue('step-3', 'up');" 
+            type="button">
+        </button>
+</div>
+<br><br>
+<h3>RTL Support</h3>
+<div dir="rtl">
+    <div class="fd-step-input">
+            <button aria-label="Step down" class="
+                fd-button
+                fd-button--transparent
+                fd-step-input__button
+                sap-icon--less"
+                onclick="stepInputValue('step-4', 'down');"
+                type="button">
+            </button>
+        <input class="
+                fd-input 
+                fd-input--no-number-spinner 
+                fd-step-input__input
+        " id="step-4" type="number" value="0">
+            <button aria-label="Step up" class="
+                fd-button 
+                fd-button--transparent
+                fd-step-input__button
+                sap-icon--add"
+                onclick="stepInputValue('step-4', 'up');" 
+                type="button">
+            </button>
+    </div>
+    
+</div>
+{% endcapture %}
+{% include display-component.html component=step-input-cozy %}
+
+
+## Compact Step Input
+Step Input can be used in compact mode, which is done to be used in desktop, or larger screen devices. 
```

Comment:
The Step Input should be used in compact mode when using a desktop of devices with large screens. It can be achieved by adding the `--compact` modifier to the main element as well as the button and input elements.